addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/aws-accounts-controller.js

```diff
@@ -49,6 +49,18 @@ async function configure(context) {
     }),
   );
 
+  // ===============================================================
+  //  GET /onboard (mounted to /api/aws-accounts)
+  // ===============================================================
+  router.get(
+    '/onboard',
+    wrap(async (req, res) => {
+      const requestContext = res.locals.requestContext;
+      const result = await awsCfnService.attemptOnboardAccounts(requestContext);
```

Comment:
Update this method to checkPendingAccounts and route to /api/aws-accounts/pending ?

Reply:
sure, that makes sense

Reply:
actually if we're going to roll this into the account permissions check, this change won't be needed

Reply:
just refactored this 👍